js/blockchaincom.js

```diff
@@ -930,14 +930,18 @@ module.exports = class blockchaincom extends Exchange {
          * @param {string} address the address to withdraw to
          * @param {string|undefined} tag
          * @param {object} params extra parameters specific to the blockchaincom api endpoint
+         * @param {string} params.beneficiary blockchaincom specific id of the whitelisted beneficiary, can be retrieved by calling fetchWithdrawalWhitelist()
          * @returns {object} a [transaction structure]{@link https://docs.ccxt.com/en/latest/manual.html#transaction-structure}
          */
         await this.loadMarkets ();
+        if (address !== undefined) {
+            throw new ExchangeError (this.id + ' withdraw () does not support the address argument, provide beneficiary parameter in params instead, for a whitelisted withdrawal address. These can be found by calling fetchWithdrawalWhitelist ()');
```

Comment:
Actually, we can support specifying the beneficiary in the address argument in this case? @DoctorSlimm lmk if you can revisit this once more.

Reply:
Sure thing will do